markdown/chapters/ddft/dd-vs-hate.md

```diff
@@ -345,10 +345,10 @@ is bad timing, i.e you set up for a certain type of kill, and Mage comes down
 and ruins it just before you would try to go off. Now your ability to shift
 gears may be limited, making it harder to recover.
 
-### Ashiok, Dream Render/Leonin Arbiter/Aven Mindcensor
+### Opposition Agent/Ashiok, Dream Render/Similar Others
```

Comment:
Maybe just change /Similar Others to something like "and friends"?

Reply:
Changed it as you suggested.